docs/conf.py

```diff
@@ -21,9 +21,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'Tungsten CI Dashboard'
-copyright = u'2018, Tungsten Fabric'
-author = u'Tungsten Fabric'
+project = 'ACID CI Dashboard'
```

Comment:
IMHO it should be `ACID` or `Awesome CI Dashboard` unless we are going to use recursive naming.